feat: Added commit count parsing feature

yusufkandemir opened the pull request:
Closes #10.

I selected the author name for other contributors as `[others]` because `other` and `others` are valid Github usernames, but `[others]` is not a valid one.

Example output:
```json
{
  "commits": [
    {
      "author": "rstoenescu",
      "count": "125"
    },
    {
      "author": "pdanpdan",
      "count": "16"
    },
    {
      "author": "[others]",
      "count": "120"
    }
  ]
}
```

yusufkandemir committed:
feat: Added commit count parsing feature

valasek commented:
Thank you works as defined I have just tested.

May I request 2 corrections:

1. Can you please remove from the commitCounts authors whose names are specified in the file `config.json`:
```
  "commitCountBlacklist": [
    "dependabot-preview[bot]"
  ]
```
2. Can you sort entries in the commitCounts array by a count field in decreasing order AND keeping [others] as the first entry? Example:
```
    "commitCounts": [
      {
        "author": "[others]",
        "count": 1
      },
      {
        "author": "rstoenescu",
        "count": 25
      },
      {
        "author": "pdanpdan",
        "count": 4
      }
    ]
```

yusufkandemir committed:
feat(commitCounts): sorted results

2.nd request of https://github.com/yusufkandemir/quasarian-in-action-helper/pull/12#issuecomment-577713180

yusufkandemir committed:
feat(commitCounts): Improved configuration and docblocks

Related with 1st request of https://github.com/yusufkandemir/quasarian-in-action-helper/pull/12#issuecomment-577713180
Also contains some docblock improvements.

yusufkandemir committed:
Merge branch 'master' into feat-add-commit-count-parsing

valasek commented:
Works as expected except one thing: When object Others count = 0 it should not be present,

Current result:
```
"quasarframework/quasar-ui-qoverlay": {
    "issues": [],
    "pulls": [],
    "releases": [],
    "commitCounts": [
      {
        "author": "[others]",
        "count": 0
      },
      {
        "author": "hawkeye64",
        "count": 1
      }
    ]
  },
  "quasarframework/app-extension-qactivity": {
    "issues": [],
    "pulls": [],
    "releases": [],
    "commitCounts": [
      {
        "author": "[others]",
        "count": 0
      }
    ]
  },
```

Expected result
```
  "quasarframework/quasar-ui-qflashcard": {
    "issues": [],
    "pulls": [],
    "releases": [],
    "commitCounts": [
      {
        "author": "hawkeye64",
        "count": 1
      }
    ]
  },
  "quasarframework/app-extension-qactivity": {
    "issues": [],
    "pulls": [],
    "releases": [],
    "commitCounts": []
  },
```

yusufkandemir committed:
feat: Display 'others' only when commit count >0

Requested in https://github.com/yusufkandemir/quasarian-in-action-helper/pull/12/#issuecomment-578205071

valasek commented:
Amazing work, as usual. Thank you for all Yusuf! Can me merged.
